README.md

```diff
@@ -12,6 +12,10 @@ This Repo is for creating a simple portoflio project.
 
 Simple Portfolio Project
 
+# Live Demo
+
+(https://waqaskanju.github.io/Portfolio/)
+
 ## Getting Started
 
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, and remove details of the second author on _line 61 through to 73_ of your _ReadMe_ as this project is done solely. Also, replace the text _author1_ with your full name. This will make your ReadMe appear more professional.

Reply:
Thanks for mentioning. Consider it done